v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -17,128 +17,134 @@
 
-  @TemplateParameter.BigQueryTable(
+  @TemplateParameter.Text(
       order = 1,
+      groupName = "Source",
       optional = true,
-      description = "BigQuery output table",
-      helpText =
-          "BigQuery table location to write the output to. The name should be in the format "
-              + "`<project>:<dataset>.<table_name>`. The table's schema must match input objects.")
-  String getOutputTableSpec();
+      regexes = {"[,:a-zA-Z0-9._-]+"},
+      description = "Kafka Bootstrap Server",
+      helpText = "Kafka Bootstrap Server list, separated by commas.",
+      example = "localhost:9092,127.0.0.1:9093",
+      hiddenUi = true)
+  @Override
+  String getReadBootstrapServers();
 
-  void setOutputTableSpec(String value);
+  void setReadBootstrapServers(String bootstrapServers);
 
-  @TemplateParameter.Enum(
+  @TemplateParameter.Text(
       order = 2,
-      enumOptions = {
-        @TemplateParameter.TemplateEnumOption("WRITE_APPEND"),
-        @TemplateParameter.TemplateEnumOption("WRITE_EMPTY"),
-        @TemplateParameter.TemplateEnumOption("WRITE_TRUNCATE")
-      },
-      optional = true,
-      description = "Write Disposition to use for BigQuery",
-      helpText =
-          "BigQuery WriteDisposition. For example, WRITE_APPEND, WRITE_EMPTY or WRITE_TRUNCATE.")
-  @Default.String("WRITE_APPEND")
-  String getWriteDisposition();
-
-  void setWriteDisposition(String writeDisposition);
-
-  @TemplateParameter.Enum(
-      order = 3,
-      enumOptions = {
-        @TemplateParameter.TemplateEnumOption("CREATE_IF_NEEDED"),
-        @TemplateParameter.TemplateEnumOption("CREATE_NEVER")
-      },
+      groupName = "Source",
       optional = true,
-      description = "Create Disposition to use for BigQuery",
-      helpText = "BigQuery CreateDisposition. For example, CREATE_IF_NEEDED, CREATE_NEVER.")
-  @Default.String("CREATE_IF_NEEDED")
-  String getCreateDisposition();
+      regexes = {"[,a-zA-Z0-9._-]+"},
+      description = "Kafka Topic(s)",
+      helpText = "Kafka topic(s) to read input from.",
+      example = "topic1,topic2",
+      hiddenUi = true)
+  @Override
+  String getKafkaReadTopics();
 
-  void setCreateDisposition(String createDisposition);
+  void setKafkaReadTopics(String inputTopics);
 
-  @TemplateParameter.BigQueryTable(
-      order = 4,
-      optional = true,
-      description = "The dead-letter table name to output failed messages to BigQuery",
-      helpText =
-          "BigQuery table for failed messages. Messages failed to reach the output table for different reasons "
-              + "(e.g., mismatched schema, malformed json) are written to this table. If it doesn't exist, it will"
-              + " be created during pipeline execution. If not specified, \"outputTableSpec_error_records\" is used instead.",
-      example = "your-project-id:your-dataset.your-table-name")
-  String getOutputDeadletterTable();
+  @TemplateParameter.KafkaTopic(
+      order = 1,
```

Comment:
Order 1 is repeated 2 times. idk the impact but lets make it unique

Reply:
Its fine, since the other one is hidden anyway.